wdl2cwl/tests/cwl_files/smoove.cwl

```diff
@@ -25,6 +27,11 @@ requirements:
             --fasta $(inputs.referenceFasta.path) \
             $(inputs.bamFile.path)
   - class: InlineJavascriptRequirement
+hints:
+  - class: ResourceRequirement
+    ramMin: 15360
+  - class: ToolTimeLimit
+    timelimit: 86400
```

Comment:
In the original WDL, the time limit was a user parameter with a default value, so we would do the same here

Reply:
I got that default value and converted into seconds. Is that incorrect?

Reply:
`timelimit` should be an expression referring to a input with that default value, just like in the original https://github.com/common-workflow-lab/wdl-cwl-translator/pull/24/files#diff-3403794b7459aa2221c7a16897161d454dddce70494206e5009454ce46750e3dR33

Reply:
I'm not sure whether I understood correcly. ```timelimit``` should be something like $(inputs.timeMinutes) and doesn't need to be converted to seconds?

Reply:
Probably safest to keep the original default value and do the conversion using a CWL expression `$(inputs.timeMinutes  * 60)`

Reply:
Ok. Can I do the same for other requirements (eg: docker requirement) too? Use the expression instead of replacing with the default value? ```$(inputs.docker)``` instead of replacing it with the actual value.

Reply:
To answer that question, we check the standard.

https://www.commonwl.org/v1.2/CommandLineTool.html#DockerRequirement

Alas, only a static string is accepted for any of the fields in `dockerRequirement`

Reply:
Shall I convert the memory (ResourceRequirement: ramMin) requirement to an expression as well or keep it as it is?

Reply:
Yes, please also convert `ramMin` into an expression